eng/pipelines/pr.yml

```diff
@@ -0,0 +1,12 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - feature/track2
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    TestProxy: true
```

Comment:
Perhaps we should set these as the default in this repo and not need to set them here.

Reply:
Set `TestProxy: true` as default.   I'm not sure we need the 2 hour test timeout.  Using the existing default of 60 minutes